patches/chrome-browser-ui-webui-settings-import_data_handler.cc.patch

```diff
@@ -10,10 +10,10 @@ index a2d1ae566df6352f21c0a437ffd5e30ba7b1abdc..a5c818276daf6376184da4f7d5e78f03
  
    const importer::SourceProfile& source_profile =
        importer_list_->GetSourceProfileAt(browser_index);
-@@ -210,6 +211,7 @@ void ImportDataHandler::SendBrowserProfileData(const std::string& callback_id) {
-     browser_profile->SetBoolKey(
-         "autofillFormData",
-         (browser_services & importer::AUTOFILL_FORM_DATA) != 0);
+@@ -207,6 +208,7 @@ void ImportDataHandler::SendBrowserProfileData(const std::string& callback_id) {
+                         (browser_services & importer::SEARCH_ENGINES) != 0);
+     browser_profile.Set("autofillFormData",
+                         (browser_services & importer::AUTOFILL_FORM_DATA) != 0);
 +    BRAVE_SEND_BROWSER_PROFILE_DATA
```

Comment:
this would probably less prone to conflicts if we put it right above `browser_profiles.Append(std::move(browser_profile));` with a space above it

Reply:
Fixed in 2f59c525b42f94fc21649400a2d79a22cf2987e9.